puma/manifest.json

```
@@ -0,0 +1,33 @@
+{
+  "display_name": "puma",
+  "maintainer": "justin.morris@ferocia.com.au",
+  "manifest_version": "1.0.0",
+  "name": "puma",
+  "metric_prefix": "puma.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "",
+  "guid": "93264c0f-a4d1-447d-81b6-bee3eb891df3",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-puma Integration",
+  "categories": [
+    ""
```

```suggestion
    "web"
```
let's include some categories. They can be found here https://docs.datadoghq.com/integrations/